spec/client_spec.rb

```diff
@@ -20,11 +20,49 @@
     subject { ActionNetworkRest::Client.new }
 
     it 'should work without a petition_id' do
-      expect(subject.petitions).to be_a ActionNetworkRest::Petitions
+      p = subject.petitions
+      expect(p).to be_a ActionNetworkRest::Petitions
+      expect(p.petition_id).to be_nil
     end
 
     it 'should work with a petition_id' do
-      expect(subject.petitions(123)).to be_a ActionNetworkRest::Petitions
+      p = subject.petitions(123)
+      expect(p).to be_a ActionNetworkRest::Petitions
+      expect(p.petition_id).to eq 123
+    end
+
+    it 'should work with two calls in a row with different arguments' do
+      p = subject.petitions
+      expect(p).to be_a ActionNetworkRest::Petitions
```

Comment:
Is this valid, with no ID?

Reply:
Yep, this is used for `client.petitions.create({some: 'data'})` calls